Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -88,10 +88,15 @@ const KeyboardAvoidingView = React.createClass({
 
   relativeKeyboardHeight(keyboardFrame: ScreenRect): number {
     const frame = this.frame;
+    const {bottom} = this.state;
```

Comment:
@brentvatne, yes, it is... do you want me to revise the whole file to follow the standard?